base/collection_n1ql_common.go

```diff
@@ -123,7 +123,7 @@ func (im *indexManager) GetAllIndexes() ([]gocb.QueryIndex, error) {
 	return im.cluster.GetAllIndexes(im.bucketName, opts)
 }
 
-// CreateIndex issues a CREATE INDEX query in the current bucket, using the form:
```

Comment:
Do you want to change example also? This might be pedantic since this will be bucket.scope.collection

Reply:
It's still a valid example if using the default collection and it doesn't seem necessary to change.